core/quill.js

```diff
@@ -84,9 +85,6 @@ class Quill {
     if (this.options.readOnly) {
       this.disable();
     }
-    if (this.options.placeholder) {
```

Comment:
> I think the CSS rules will work either way.

They will, but I think it's important to ensure a logical order.
What about moving it right after the `this.root` initialization?